bungiesearch/signals.py

```diff
@@ -17,32 +18,39 @@ def get_signal_processor():
         signal_class = BungieSignalProcessor
     return signal_class()
 
-__items_to_be_indexed__ = defaultdict(list)
```

Comment:
Check this doc out https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming
that's where I learn a lot of my style guides from, Google's various style guides for different languages